keras_nlp/tokenizers/tokenizer.py

```diff
@@ -125,4 +125,4 @@ def call(self, inputs, *args, training=None, **kwargs):
         return self.tokenize(inputs, *args, **kwargs)
 
     def compute_output_shape(self, inputs_shape):
```

Comment:
Actually I think we should just remove the whole function here. It is not the case that our input shape == our output shapes. We are just removing the extra bit of functionality to compute the output shape without running `call`, which is something we probably do not need for tokenizers.

Reply:
I have removed the compute_output_shape method